server/Gopkg.toml

```diff
@@ -3,10 +3,10 @@
   go-tests = true
   unused-packages = true
 
-[[constraint]]
-  name = "github.com/mattermost/mattermost-server"
-  version = "~5.6.0"
-
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "~1.2.0"
+
```

Comment:
Looks like you added a couple of dependencies. What do you think about adding these in this file explicitly?